pkg/connector/sso_group.go

```diff
@@ -236,12 +239,23 @@ func (g *ssoGroupResourceType) createOrGetMembership(
 	}
 	foundMembership, err := g.identityStoreClient.GetGroupMembershipId(ctx, &getInput)
 	if err != nil {
-		return nil, err
+		// If we lack permission for the `GetGroupMembershipId` operation, fail
+		// more gracefully by returning nil.
+		if strings.Contains(
```

Comment:
future TODO: aws sdk you can extract the AWS Error Type as a type instead of a string match